client/src/common/definitions/union/IPartyData.ts

```diff
@@ -0,0 +1,20 @@
+export interface IPartyData {
+  users?: IUserData[];
+}
+
+export interface IUserData {
+  username: string;
+  data: IUserGrailData;
+}
+
+export interface IUserGrailData {
+  uniqueArmor?: ILBStatsData;
+  uniqueWeapons?: ILBStatsData;
+  uniqueOther?: ILBStatsData;
+  sets?: ILBStatsData;
+  itemScore?: number;
+}
+
+export interface ILBStatsData {
```

Comment:
`LBS`?

Reply:
This is an abbreviation => LeaderBoardStats = LBStats. I'm not sure why I abbreviated this...will fix on next commit